<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 20, 20);
font: 700 26pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>User Panel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableResults">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>163</y>
     <width>661</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>661</width>
     <height>371</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>661</width>
     <height>371</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>321</number>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search Patient Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-5</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/user/image/user.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>labelTitle</zorder>
  <zorder>tableResults</zorder>
  <zorder>btnLogout</zorder>
  <zorder>txtSearch</zorder>
  <zorder>btnSearch</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
